src/sentry/models/outbox.py

```diff
@@ -52,6 +53,7 @@ class OutboxCategory(IntEnum):
     VERIFY_ORGANIZATION_MAPPING = 4
     AUDIT_LOG_EVENT = 5
     USER_IP_EVENT = 6
+    INTEGRATION_UPDATE = 7
```

Comment:
Categories, however, define *what* sort of thing happened.  Usually it is an update, or an event, associated with the scope that the category combines with.  The category decides which action is taken by the receiver when being processed.  It also determines which events can be aggregated together.  Entries with the same identifier and category are coalesced together, only the most recent being sent.